src/main/java/school/lemon/changerequest/java/introduction/hw1/Task6.java

```diff
@@ -1,11 +1,18 @@
 package school.lemon.changerequest.java.introduction.hw1;
 
 public class Task6 {
-    public static double calculateS(double x) {
-        return 0d;
-    }
+    public static double calculateS(double x)
+    {
+        int f1=2;
+        int f2=2*3;
+        int f3=2*3*4;
+        return
```

Comment:
All my attempts to make the method failed. I think we,re study it in next lesson.